Pipfile

```diff
@@ -42,32 +76,17 @@ dill = "==0.3.8"
 
 [requires]
 python_version = "3.11"
+#python_full_version = "3.11.8"
 
 [scripts]
-cli = "python run.py"
-cli-ext = "bash -c \"python $(dirname \\$PIPENV_PIPFILE)/src/mpyl/__main__.py $(echo $0) $(echo $@)\""
```

Comment:
This is also still needed for running from the monorepo right?

Reply:
I couldn't find [any ocurrences of `cli-ext` in the monorepo](repo:Vandebron/Vandebron cli-ext), so I guess not.

Reply:
I sometimes still want to quickly run gh-mpyl locally without having to build the image every time, that's what this command is used for. There's no harm in keeping it right?

Reply:
I usually run `pipenv run cli …`, is there a difference?